editor/src/clj/editor/scene.clj

```diff
@@ -1237,9 +1238,16 @@
 
 (defn- nudge! [scene-node-ids ^double dx ^double dy ^double dz]
   (g/transact
-    (for [node-id scene-node-ids
-          :let [[^double x ^double y ^double z] (g/node-value node-id :position)]]
-      (g/set-property node-id :position [(+ x dx) (+ y dy) (+ z dz)]))))
+   (for [node-id scene-node-ids
+         :let [[^double x ^double y ^double z] (g/node-value node-id :position)
+               current-layout (g/maybe-node-value node-id :current-layout)
+               new-position [(+ x dx) (+ y dy) (+ z dz)]]]
+     (if (str/blank? current-layout)
+       (g/set-property node-id :position new-position)
+       (g/update-property
+        node-id :layout->prop->override
+        update current-layout
+        assoc :position new-position)))))
```

Comment:
We should probably make `gui/update-layout-property` public and call it here instead of re-implementing it?

Style node: We use two spaces for indentation.

Reply:
We could use `g/maybe-node-value` in `gui/update-layout-property` without breaking things. However, thinking about it some more, I'm starting to think maybe `nudge!` should simply call `scene-tools/manip-move`, since it already has a specialization for `GuiNode` that respects the layout property. 🤔

Reply:
That seems to work. The evaluation-context is not required in this case (as far as I understand), but I don't think that is a problem.